third-party/linshare/src/main/java/org/apache/james/linshare/LinshareConfiguration.java

```diff
@@ -30,67 +30,82 @@
 
 public class LinshareConfiguration {
 
-    public static final String URL = "blob.export.linshare.url";
-    public static final String TOKEN = "blob.export.linshare.token";
+    public static final String URL_PROPERTY = "blob.export.linshare.url";
+    public static final String USER_NAME_PROPERTY = "blob.export.linshare.user.name";
+    public static final String PASSWORD_PROPERTY = "blob.export.linshare.user.password";
```

Comment:
We don't use a `user` but a `technicalAccount`. Please adapt both properties accordingly.